ext/mbstring/mbstring.stub.php

```diff
@@ -118,7 +118,6 @@ function mb_strcut(string $string, int $start, ?int $length = null, ?string $enc
 
 function mb_strwidth(string $string, ?string $encoding = null): int {}
 
-/** @refcount 1 */
```

Comment:
I guess this hint is used for some kind of optimizations by the JIT? Probably for cases where the returned string doesn't "escape" and can be immediately freed at the end of a block?

I just looked through the entire list and the annotations look accurate for the other functions. There are some other cases where potentially we *could* just return the input string unmodified, which might help to reduce the number of dynamic allocations. If I ever do that I will remember to adjust the annotations here.

Reply:
I do wonder why `mb_scrub` isn't annotated as `@refcount 1`. I believe that function should always return a newly allocated string with refcount 1. Or... is there something I'm missing?